packages/terra-menu/src/_MenuWidths.js

```diff
@@ -0,0 +1,10 @@
+const MenuWidths = {
```

Comment:
I'm using a hash for this so that I don't have to convert between string and number, since the popup api requires a string and I need the number for fullscreen calculations.